build/deb/debian/copyright

```diff
@@ -0,0 +1,35 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: milvus
+Source: https://github.com/milvus-io/milvus
+
+Files: *
+Copyright: 2019-2021, Zilliz <yunmei,li@zilliz.com>
```

Comment:
is it okay to leave a personal mail in copyright? and shall we update the copyright to 2022